tests/designs/array_buses/Makefile

```
@@ -0,0 +1,24 @@
+TOPLEVEL_LANG ?= verilog
+
+ifneq ($(TOPLEVEL_LANG),verilog)
+
+all:
+	@echo "Skipping test due to TOPLEVEL_LANG=$(TOPLEVEL_LANG) not being verilog"
+clean::
+
+else
+ifeq ($(TOPLEVEL_LANG),verilog)
+   TOPLEVEL := array_buses
+
+   PWD=$(shell pwd)
+
+   COCOTB?=$(PWD)/../../..
+
+   VERILOG_SOURCES = $(COCOTB)/tests/designs/array_buses/array_buses.sv
```

We no longer use `COCOTB`
```suggestion
   VERILOG_SOURCES = $(PWD)/array_buses.sv
```